packages/@ember/-internals/glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2492,7 +2492,7 @@ moduleFor(
           },
           value: null,
         }),
-        template: '<div id="inner-value">{{wrapper.content}}</div>',
+        template: '<div id="inner-value">{{this.wrapper.content}}</div>',
```

Comment:
Something is fishy/scary that these continued to work, seemingly without assertions/deprecations either. Hopefully the new VM code is erroring consistently now, but this means we may have implicit this code in the wild that was accidentally working

Reply:
I think `expectAssertion` below would have swallowed any other assertions, even if it matched on `backtrackingMessageFor`